packages/app-info/README.md

```diff
@@ -49,7 +49,12 @@ For AWS Lambda, you can use a plugin like [serverless-plugin-git-variables](http
 
 ### `appInfo.environment`
 
-Get the application environment, normally either `development` or `production`. This will be a string, using `process.env.NODE_ENV` and defaulting to `development`.
+Get the application environment, normally either `development` or `production`. This will be a string, trying each of the following environment variables in order and defaulting to `development`:
```

Comment:
Right now we're not doing any normalisation here but I'm not against adding this in future if you want consistent values, @sjparkinson? E.g. we could convert `p` and `prod` to `production` automatically

Reply:
Not fussed! Great if it's aligned but we could also do that in the Gateway Collector if we want consistency across the board.

Reply:
As a note, the values the Gateway supports for "production are as you listed, `p`, `prod` or `production` nothing else.

https://github.com/Financial-Times/observability/blob/aeb21e682ebfcaefbefa6501705c9645a523c130/services/otel-gateway-collector/config/otel-collector-pipeline-otlp.yaml#L42-L47